metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -27,7 +27,7 @@
        ;; fully populated.
        "granule_ur VARCHAR(250),
         transaction_id INTEGER DEFAULT 0 NOT NULL,
-        created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL"))
+        created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP"))
```

Comment:
I like the old way. I don't think we should make our db schema worse just to be consistent with existing tables.

Reply:
Done.